README.md

```diff
@@ -43,14 +43,36 @@ Default config:
 # config/config.exs
 config :coney,
   adapter: Coney.RabbitConnection,
-  pool_size: 1,
   auto_start: true,
   settings: %{
     url: "amqp://guest:guest@localhost", # or ["amqp://guest:guest@localhost", "amqp://guest:guest@other_host"]
     timeout: 1000
   }
 ```
 
+If you need to create exchanges or queues before starting the consumer, you can define your RabbitMQ topology as follows:
+```elixir
+config :coney,
+  topology: %{
+    exchanges: [{:topic, "my_exchange", durable: true}],
+    queues: [
+      %{
+        name: "my_queue",
```

Comment:
what do you think if we change queues list to map?

```
%{
  "my_queue" => %{
    options: [],
    bindings: []
  }
}
```

Reply:
Sure, for aesthetics or is there another reason?

Reply:
just to make it shorter